docs/src/dev-guide/design-clp-structured/single-file-archive-format.md

```
@@ -0,0 +1,354 @@
+The ArchiveInfo packet is a msgpack map that currently only records the number of segments in an
+archive as shown in [Figure 4](#figure-4). Each archive currently consists of only a single segment
+(i.e. there is only one tables segment file), but we still record the number of segments in order
+to offer backwards compatibility if we do start splitting tables into multiple segments.
+
+This metadata packet was originally intended to mimic the "ArchiveMetadata" structure in CLP, but
+most of what "ArchiveMetata" records is now present in either the header or the RangeIndex metadata
+packet.
+
+(figure-4)=
+::::{card}
+```json
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)